pom.xml

```diff
@@ -207,13 +208,20 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.19.1</version>
+        <version>3.2.5</version>
         <configuration>
           <!-- Skips unit tests if the value of skip.unit.tests property is true -->
           <skipTests>${skip.unit.tests}</skipTests>
+          <includes>
+            <include>**/*Test</include>
+            <!--  Run all spock test except Repose-->
+            <include>**/*Spec</include>
+          </includes>
           <!-- Excludes integration tests when unit tests are run. -->
+          <!-- Excludes spock tests when unit tests are run. -->
           <!-- This is only necessary because the failsafe plugin's test naming conventions are overridden. -->
```

Comment:
You added a comment in between two related comments. Now it looks like there's something about spock and failsafe, which is incorrect.

Reply:
Updated